pkg/aws/policies.go

```diff
@@ -841,6 +841,10 @@ func (c *awsClient) mapToAccountRoles(version string, roles []iamtypes.Role) ([]
 			return accountRoles, err
 		}
 
+		if !checkIfAccountRole(role.RoleName) {
+			return []Role{}, nil
```

Comment:
I dont think this has the intended outcome we want. It will always return empty list if it finds one role that is not an account role. 

I would think maybe something like 
```
var accountRoles []Role
	for _, role := range roles {
		if !checkIfAccountRole(role.RoleName) {
			continue
		}
		accountRole, err := c.mapToAccountRole(version, role)
		if err != nil {
			return accountRoles, err
		}
		accountRoles = append(accountRoles, accountRole)
	}
	
If we move the check before the map to role, and just continue if its not an account role, should do the trick

Reply:
you're right thanks for the spot